pkg/operator/bootstrap.go

```diff
@@ -101,7 +101,6 @@ func RenderBootstrap(
 	spec.OSImageURL = imgs.MachineOSContent
 	spec.Images = map[string]string{
 		templatectrl.EtcdImageKey:            imgs.Etcd,
-		templatectrl.SetupEtcdEnvKey:         imgs.SetupEtcdEnv,
```

Comment:
Basically, we can't just remove this for templates under `templates/` otherwise how https://github.com/openshift/machine-config-operator/pull/895/files#diff-db87ea55a6cbf22e6bd99d79dc77e943R16 knows about the MCO image itself (see https://github.com/openshift/machine-config-operator/pull/895/files#r298159078)

Reply:
ok gotcha. this one was trickier than the others and i wasn't 100% sure on it. thanks!

Reply:
i think bc the other times i passed the MCO image with the command override I wasn't totally sure here about whether this is a delete or a replace with MCO image. Willl update!!